pkgs/applications/networking/browsers/chromium/default.nix

```diff
@@ -161,7 +161,7 @@ in stdenv.mkDerivation {
 
   buildCommand = let
     browserBinary = "${chromiumWV}/libexec/chromium/chromium";
-    libPath = lib.makeLibraryPath [ libva pipewire_0_2 ];
```

Comment:
If we do not have any other use for pipewire_0_2 it should go.

Reply:
From a quick git-grep it looks like Mutter still requires it:
```
pkgs/desktops/gnome-3/core/mutter/3.34/default.nix:, pipewire_0_2
pkgs/desktops/gnome-3/core/mutter/3.34/default.nix:    pipewire_0_2 # TODO: backport pipewire 0.3 support
```

Reply:
cc @jtojnar is this still the case?

Reply:
Yup, the old version of Mutter used by Pantheon uses 0.2. The current version used by GNOME uses 0.3, obviously.